src/client/src/containers/RightSidebar/index.tsx

```diff
@@ -73,6 +74,13 @@ type Props = IRightSidebarProps &
   IDispatchProps &
   InjectedIntlProps;
 
+const hasAzureServices = (services: any) => {
+  for (var key in services) {
```

Comment:
This PR has made me realize it would be much simpler with getServicesSelector from the wizardSelectionSelector file as this would have been a simple .length > 0. Should probably do that sometime.

Anyways, most nitpicky thing I can say is use const instead of var.

Could maybe even do
```ts
 Object.values(services).some(service => !_.isEmpty(service.selection));
```
Or something of the sort, but its good as it is now too.

Reply:
Yes! It's not good, var is bad, I'll fix it :D